components/member-profile/index.js

```diff
@@ -1,29 +1,29 @@
-import { getDataURL, getImgURL } from 'helper-functions/urls';
-
 import PropTypes from 'prop-types';
 import React from 'react';
 import classNames from './member-profile.module.scss';
-import useFetch from 'custom-hooks/useFetch';
 
 const Profile = (props) => {
-  const imgLink = getImgURL(props.id);
-  const { data } = useFetch(getDataURL(props.id));
+  const memberName = `${props.membersData.first_name || '--'} ${
+    props.membersData.last_name || '--'
+  }`;
```

Comment:
Is there any specific reason to do this?
Also, you can use destructure props. 
```
const { memberdData, imageLink } = props;
```

Reply:
Yes ok I will destruct the props.
The reason why I am displaying in this way is because shihab once said that what if no first_name or last_name exists.

Reply:
Yes, it is always good to have fallback values. But here you can do something like,

```
const { membersData:{first_name, last_name} }=props; // de-structure all props on top
const memberName=`${first_name||''} ${last_name||''}`;  // to avoid -- when either one is missing
```

while accessing `{memberName.trim()||'--'}`

_On a general note_
accessing values like `props. membersData.first_name` is easy to break. So better to use it like `props. membersData?.first_name`

Reply:
Got It, Thank you